packages/tempus/src/Tempus_StepperFactory.hpp

```diff
@@ -1459,56 +1496,58 @@ class StepperFactory
     std::vector<Teuchos::RCP<const Thyra::ModelEvaluator<Scalar> > > appModels,
     Teuchos::RCP<Teuchos::ParameterList> stepperPL)
   {
-    using Teuchos::RCP;
-    using Teuchos::ParameterList;
+    if (stepperPL != Teuchos::null) {
+      using Teuchos::RCP;
+      using Teuchos::ParameterList;
+
+      // Parse Stepper List String
+      std::vector<std::string> stepperListStr;
+      stepperListStr.clear();
+      std::string str = stepperPL->get<std::string>("Stepper List");
+      std::string delimiters(",");
+      // Skip delimiters at the beginning
+      std::string::size_type lastPos = str.find_first_not_of(delimiters, 0);
+      // Find the first delimiter
+      std::string::size_type pos     = str.find_first_of(delimiters, lastPos);
+      while ((pos != std::string::npos) || (lastPos != std::string::npos)) {
+        std::string token = str.substr(lastPos,pos-lastPos);
+        // Strip single quotes
+        std::string::size_type beg = token.find_first_of("'") + 1;
+        std::string::size_type end = token.find_last_of ("'");
+        stepperListStr.push_back(token.substr(beg,end-beg));
+
+        lastPos = str.find_first_not_of(delimiters, pos); // Skip delimiters
+        pos = str.find_first_of(delimiters, lastPos);     // Find next delimiter
+      }
```

Comment:
Not saying we need to do this now, but I have similar code in panzer. One day we should move things like this into a teuchos string utils library.

Reply:
Yeah, you shared it with me originally from panzer.  Putting it in teuchos would be good.